mllib/src/main/scala/org/apache/spark/mllib/regression/LinearRegression.scala

```diff
@@ -141,7 +143,7 @@ object LinearRegressionWithSGD {
       stepSize: Double,
       miniBatchFraction: Double,
       initialWeights: Vector): LinearRegressionModel = {
-    new LinearRegressionWithSGD(stepSize, numIterations, miniBatchFraction)
+    new LinearRegressionWithSGD(stepSize, numIterations, 0.0, miniBatchFraction)
```

Comment:
This is the same way with `LogisticRegressionWithSGD`. The signature of `train` function is not changed.